### PR TITLE
[Industry Core KIT 24.08] Updated Software Development View

### PR DESCRIPTION
**Pull request by johannsvarela**
## Description
Added new guidelines into the Digital Twin section of the Software Development View. For more details have a look at the updated changelog.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by johannsvarela** [comment]
@mhellmeier i can't assigne you therefore i'll try it this way. Please review my PR.

**Comment by arnoweiss** [comment]
Please fix the [`DEPENDENCIES` file](https://github.com/[eclipse-tractusx/eclipse-tractusx.github.io](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io?tab=readme-ov-file#3rd-party-dependency-checks)?tab=readme-ov-file#3rd-party-dependency-checks) to make the pipeline pass.

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx [hunk 1]
```diff
@@ -38,6 +38,14 @@ In Industry Core, different types of parts, e. g. serialized parts, batches, JIS
 
 To enforce a strict need-to-know principle (and to prevent data from being exposed to non-authorized partners in the Catena-X network), the visibility of digital twins and the content of the attribute `specificAssetIds` of digital twins must be restricted to authorized partners only. The actual implemenetation depends on the DTR product used by the Catena-X partner. More details can be found in the [Digital Twin KIT](../../../../../docs-kits/category/digital-twin-kit).
 
+#### Handling of Digital Twins at Contract Manufactoring
```
Comment (arnoweiss) [inline docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:41]:
```suggestion
#### Handling of Digital Twins at Contract Manufactoring
```

**Review by arnoweiss — APPROVED** [review]

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx (hunk 1 above)
Comment (johannsvarela) [reply docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx]: @arnoweiss Done!

**Review by arnoweiss — APPROVED** [review]

**Review by mhellmeier — APPROVED** [review]

**Pull request merged**